test/app/README.md

```diff
@@ -0,0 +1,12 @@
+# test-app
+
+Install dependencies
+
+`npm i`
+
+Set or pass environment variables.
+Can be set locally using a [.env file](https://github.com/motdotla/dotenv#usage)
+
+`CLIENT_ID=XXX
+DOMAIN=xxx.okta.com
+npm start`
```

Comment:
The root repo uses `yarn` as our dependency manager. Do you think its confusing to include a `package-lock` for the embedded test application, but a `yarn.lock` for the primary project?

Reply:
honestly, I'd prefer not to checkin any .lock file here. But I do not want anything hoisted from the test app to the parent directory. They are not/should not be connected in any way. This app is a test driver. Possibly the hoisting only comes into play for yarn 'workspaces'. Mostly I am trying to keep the test app as simple as possible.

Reply:
FYI, `npm install` in this directory will not be called as part of the test/build process. The test app (as a standalone project) is only there for local developers to play with. The code for the testApp is used in karma test, but this is part of main project's package.json.

Reply:
Makes sense. FWIW we found that when we mixed `npm` and `yarn` it created too much confusion for external developers. This is the reason why all of our other JS projects leverage and encourage yarn for interfacing with `npm scripts` (`yarn install`, `yarn start`, etc).

> honestly, I'd prefer not to checkin any .lock file here. 

Since this isn't used for CI - I'd suggest removing it entirely.

Reply:
did remove the lock file for test app